### ISSUE TITLE
Blank window

### ISSUE THREAD

**Issue by AhmadShamli** [jeanpaulrichter/nppcrypt#7]
Hi,

I'm getting blank Hash window,and other windows as listed below,
![image](https://user-images.githubusercontent.com/1076016/33157766-6bdcd94e-d03f-11e7-931e-848178e56b21.png)
![image](https://user-images.githubusercontent.com/1076016/33157770-72a323d2-d03f-11e7-9eda-b5179c07dcb5.png)
![image](https://user-images.githubusercontent.com/1076016/33157772-7772ebb8-d03f-11e7-9035-67fcacb44993.png)
![image](https://user-images.githubusercontent.com/1076016/33157777-8007d4dc-d03f-11e7-8931-afcbf48d7ee7.png)
![image](https://user-images.githubusercontent.com/1076016/33157781-864f8b82-d03f-11e7-807b-91b753ce9fd9.png)
![image](https://user-images.githubusercontent.com/1076016/33157785-8cf84942-d03f-11e7-80db-f97760f13e05.png)

Notepad++ version:
![image](https://user-images.githubusercontent.com/1076016/33157743-3dbe1726-d03f-11e7-874c-769df2c5cf97.png)

NppCrypt version:
![image](https://user-images.githubusercontent.com/1076016/33157754-4a867f3e-d03f-11e7-9897-0701ac46f189.png)

Current list of plugin:
![image](https://user-images.githubusercontent.com/1076016/33157832-fab8ba8e-d03f-11e7-8683-3f0bd2bed87e.png)

**Comment by jeanpaulrichter** [comment]
hi. this is very strange. at the moment i am not able to replicate it, but i'm looking into it. sorry for not having an answer right away :/

**Comment by yulinsoft** [comment]
我也出现这个现象。

**Comment by fabrice24000** [comment]
I have the same problem under Windows 7 pro but not on my old computer under 7 integral edition SP1 and i think it's since your last update because i didn't have this problem before.

**Comment by RedDeadElite** [comment]
I have the same issue, even with all plugins removed except nppcrypt. 
I'm using notepad++ v7.5.6 (32bit) with nppcrypt v1.0.1.5 x86. 
My OS is Windows 7 Pro 64bit, if that matters.

**Comment by jeanpaulrichter** [comment]
@RedDeadElite i'm still not sure what causes this problem with dockable dialogs for some users. when you find the time two things might help me: 
1. do you experience any problems with notepad++ own function list (view->function list)?
2. could you try https://owncloud.schi-jo.de/index.php/s/Pfxjwhi3Ph6PATh ? to be honest i don't think this will solve it, but just to be sure... [please use this version only for a short test and remove it afterwards]

if i cannot solve this I will just make them normal dialogs again in the next version. i'm not a huge fan of this dockable feature in notepad++ anyway...

**Comment by RedDeadElite** [comment]
@jeanpaulrichter Thanks for responding.

1. I don't have a problem with the function list, besides the (hopefully, unrelated) issues mentioned at https://github.com/notepad-plus-plus/notepad-plus-plus/issues/2924 and https://notepad-plus-plus.org/community/topic/12697/function-list-not-working-in-v7-2
2. This didn't work either.

**Comment by jeanpaulrichter** [comment]
@RedDeadElite thank you!
no, i don't think the issues are in any way related. it's really annoying that for the life of me i cannot replicate this problem despite multiple people reporting it. i tried i.e. win7 on a virtual machine without any luck. anyways, two more small things if you don't mind: the "conversion panel" of donho's "converter" plugin works also fine for you i assume? and here is another quite aimless stab in the dark ^^ : https://owncloud.schi-jo.de/index.php/s/zYJTgCeOdaLSVM9

**Comment by RedDeadElite** [comment]
@jeanpaulrichter 
The conversion panel of Don Ho's conversion plugin is working, no problem.
Test2 gives the same results as the first test.

**Comment by anton-arnold** [comment]
@jeanpaulrichter I can report this behavior too on a Win7 64bit installation
![nppcrypt-empty-window](https://user-images.githubusercontent.com/43671844/51841427-e8936980-230e-11e9-94a8-eee3bf526864.png)

Purged installation and update to latest Notepad++ 64Bit didn't fix the problem
![nppcrypt-empty-window-2](https://user-images.githubusercontent.com/43671844/51841788-b33b4b80-230f-11e9-9171-1145b1b4f0d8.png)

Dragging and moving around the working nppcrypt windows works smoothly.
The empty nppcrypt windows move much slower and seem to increase CPU load.
I can dock the empty windows below the status bar while this is not possible e.g. for crypt/encrypt.

Finally by messing around with the dialogs I managed to crash everything but then new dialog elements appeared
![nppcrypt-empty-window-crash](https://user-images.githubusercontent.com/43671844/51842354-f944df00-2310-11e9-92b4-c2909ead8fa4.png)

I tried to attach a x64 debugger to get additional information but wasn't able to get reliable information.
The process might have been stuck in a deadloop caused by WinAPI UI element callbacks.
